app/scripts/lib/buy-eth-url.test.js

```diff
@@ -12,7 +13,7 @@ describe('buy-eth-url', function () {
   const mainnet = {
     chainId: MAINNET_CHAIN_ID,
     amount: 5,
-    address: '0x0dcd5d886577d5081b0c52e242ef29e70be3e7bc',
+    address: '0x0dcd5d886577d5581b0c524242ef2ee70be3e7bc',
```

Comment:
Tiny suggestion but this address appears all over this test file -- might want to make it a constant and use that throughout.